client/src/static/mockedDataDemo.ts

```diff
@@ -31,7 +32,8 @@ export const buildInitialModelsList = ({ SIR_PN, SIR_FN, SEIR_PN, SEIRD_PN, SIRD
       },
       modelGraph: [
         {
-          file: '',
+          donuType: Donu.Type.GROMET_PNC,
+          model: 'sir.gromet',
```

Comment:
@RosaRomeroGomez this is what I was referring to in [Slack ](https://unchartedsoftware.slack.com/archives/C01GHFTFC4W/p1626119307034100).
I infered which model name from [Donu API `list-model`](https://observablehq.com/@adamocarolli/donu-test-1?collection=@adamocarolli/donu) response matched our mock data for the demo.